org-node/src/client.rs

```diff
@@ -421,13 +424,10 @@ impl Client {
                 // we want to set the head.
                 Ok(ProtocolEvent::Gossip(box Gossip::Put {
                     payload: Payload { urn, .. },
-                    result: Applied(Payload {rev, origin, .. }),
+                    result: Applied(Payload {rev: Some(Rev::Git(oid)), origin, .. }),
```

Comment:
```suggestion
                    result: Applied(Payload {rev: Some(Rev::Git(oid)), origin: Some(origin), .. }),
```

We don't handle the `None` case anyway, so may aswell.